repository_browser/lib/main.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:flutter/material.dart';
+import 'package:http/http.dart' as http;
+import 'dart:convert' as convert;
+
+void main() {
+  runApp(MyApp());
+}
+
+class Repository {
```

Comment:
Can you move this to a file called `repository.dart`? And import it into main.dart.